src/App.css

```diff
@@ -1,39 +1,263 @@
-.App {
+:root {
+  --Montserrat: 'Montserrat', sans-serif;
+  --RobotoSlab: 'Roboto Slab', serif;
+  --azure: #0290ff;
+  --z: #fafafa;
+  --white: #e8e8e8;
+  --size: 6rem;
+  --fg: #0290ff;
+  --bg: rgb(211, 211, 211);
+  --value: 64;
+  --white-two: #fff;
+  --cool-blue: #4386bf;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  background-color: #f5f6fa;
+}
```

Comment:
Great design @bhushan354 🤝

- [ ] As stated in the project requirements, kindly ensure your design replicates the provided design. Kindly remove backgrounds from all your texts so that is can match the design given.
- [ ] Also ensure the colors of your fonts match the ones provided in the design.
- Your design
![your texts](https://github.com/bhushan354/BookStore/assets/63917725/842ebc05-a0fc-4879-b395-c6831adc6e93)
- Given design
![given texts](https://github.com/bhushan354/BookStore/assets/63917725/fe5d6bc9-724c-4ab5-90e9-4adc71bda846)